packages/react-client/src/trackManager.ts

```diff
@@ -62,11 +65,11 @@ export const useTrackManager = <PeerMetadata, TrackMetadata>({
     mediaManager?.stop();
   }
 
-  async function startStreaming(
-    trackMetadata?: TrackMetadata,
-    simulcastConfig?: SimulcastConfig,
-    maxBandwidth?: TrackBandwidthLimit,
-  ) {
+  function getType(): TrackMetadata["type"] {
+    return TRACK_TYPE_TO_DEVICE[mediaManager.getDeviceType()];
+  }
```

Comment:
would it be simpler to write just if/switch statement here?

Reply:
I've implemented this, as @czerwiukk said in [this](https://github.com/fishjam-cloud/web-client-sdk/pull/69#discussion_r1731092311) comment. I don't have any preferences. Map is okej for me.